### PR TITLE
ssn form

### PR DESCRIPTION
**Pull request by daviddkkim**

### CODE REVIEW

**Review by mrestuccia — APPROVED** [review]

File: pages/index.tsx [hunk 1]
```diff
@@ -248,6 +249,17 @@ function Home() {
                 }}
               />
             </FieldSet>
+            <FieldSet>
```
Comment (mrestuccia) [inline pages/index.tsx:252]: I think we have an SSN mask component?

**Review by mrestuccia — APPROVED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (daviddkkim) [reply pages/index.tsx:252]: oooh do you have a link

Comment (mrestuccia) [reply pages/index.tsx:252]: No. @mary-snow do you know if we have a react mask component?

Comment (mary-snow) [reply pages/index.tsx:252]: I did an ssn mask for prove prefill! You can check it out here and there's some helpers used that come from a helper file. https://github.com/UseAlloy/Alloy/blob/master/sdk/frontend/src/containers/Prove/Prefill/index.tsx#L93

Comment (daviddkkim) [reply pages/index.tsx:256]: added!

**Review by philip-fahim — APPROVED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (mary-snow) [reply pages/index.tsx:256]: Wondering if you also wanted to add the `format` function to the masked ssn. So that way it's in a XXX-XX-XXXX format

**Review by mary-snow — APPROVED** [review]
Left comment, but not blocking

**Comment by mrestuccia** [comment]
@daviddkkim you got a triple check :)

File: pages/index.tsx (hunk 1 above)
Comment (daviddkkim) [reply pages/index.tsx:256]: added !

**Review by mary-snow — APPROVED** [review]

**Pull request merged**